#
# aarhus_tnu_nanopore_analysis

### One stop shop with tutorials necessary to perform DNA Nanopore Analysis

##
## Step 1 - Command line basics (Skip if you already know how to use the terminal and VIM or another command line text editor like nano): 


Before you start doing anything with Nanopore sequencing you need to learn the basics of the command line and
VIM (the text editor used in the command line). Please watch the videos below to learn the basics.

Command Line Videos: https://www.youtube.com/watch?v=s3ii48qYBxA

VIM videos: https://www.youtube.com/watch?v=wACD8WEnImo&list=PLT98CRl2KxKHy4A5N70jMRYAROzzC2a6x


##
## It is assumed that you are accessing the Desktop computer in the "DNA Lab" at Aarhus to perform Steps 2 through Step 5.



## Step 2 - The basics of Nanopore sequencing and setting up your sequencing run and basecalling.

Tutorial video by Oxford Nanopore Technologies on how to use the MinKNOW software: https://www.youtube.com/watch?v=oCOixfewXo4

my video tutorial with more information about using MinKNOW. It contains very important information, it is a must watch: https://youtu.be/Tr4hCS6DBSQ

If you want to learn more about Nanopore sequencing watch the full Nanopore Masterclass series: https://www.youtube.com/watch?v=MYOSUCgr3Y4&list=PLxpxXZj-IgXwq7AUQ93ek7zcgMjS_E0Rd

##
## Step 3 - Pulling data from GenomeDK

Video tutorial: https://youtu.be/v9ImfpTUMOE

##
## Step 4 - Getting the data that has been previously basecalled and aligned through the DNA Sequencing Pipeline for DNA modifications

Pipeline GitHub with documentation: https://github.com/bernardo-heberle/DCNL_NANOPORE_PIPELINE

Video Tutorial (Video was too long for YouTube, had to upload on Google Drive. Download the file for better video quality. If you try to watch on Google Drive it will be pixelated): https://drive.google.com/file/d/1XZsST1Onb3EWIUMg-oB9u43Z0oijpGD3/view?usp=drive_link



##
## Description of main output from DNA Sequencing Pipeline for DNA modifications

Video Tutorial: https://youtu.be/w4Ylvjt_zrU

Modkit Documentation: https://nanoporetech.github.io/modkit/quick_start.html

##
## If you want to learn more:

Learn more about NextFlow: https://www.nextflow.io/docs/latest/index.html

Many other NextFlow pipelines for all kinds of data analysis in the nfcore community: https://nf-co.re/pipelines/ - https://nf-co.re/

A different pipeline made by me to analyze RNAseq/cDNAseq Nanopore Data: https://github.com/UK-SBCoA-EbbertLab/cDNA_pipeline

Other things that will help you become a better bioinformatician and data analyst:

- Take free courses teaching command line basics and bash scripting.
- Take introductory programming courses in Python and/or R.
- Take courses for data analysis and statistics in Python and/or R.
- Learn about software containers like Apptainer/Singularity, Docker, and others.
- Learning about Zenodo and Synapse.org. these are free data repositories to share data upon publication,
- Do data analysis and bionformatics projects. Nothing build proficiency like real world problems and practice.


  ##
  ## You can find all of the powerpoints I presented during the workshop, the PDF to the publication I presented on, and downloadable video files for the tutorials I recorded in the shared Google Drive below:

  https://drive.google.com/drive/folders/1cElMLgfCq_6pnn4O0P0AomFrAS0E8C_s?usp=drive_link\

  Notice that the powerpoints contain a lot of animations, so you may need to see it in presentation mode and click through the animations to see all of the figures.
  Feel free to use any slides for your own presentations/projects.

##
##

If you get really stuck and can't find a solution after trying to troubleshoot on your own for at least 3 days feel free to contact me: bernardo.aguzzoli@gmail.com
