# aarhus_tnu_nanopore_analysis
One stop shop with tutorials necessary to perform DNA Nanopore Analysis


## Step 1 - Command line basics (Skip if you already know how to use the terminal and VIM or another command line text editor like nano): 


Before you start doing anything with Nanopore sequencing you need to learn the basics of the command line and
VIM (the text editor used in the command line). Please watch the videos below to learn the basics, this will
take several hours.

Command Line Videos: https://www.youtube.com/watch?v=s3ii48qYBxA

VIM videos: https://www.youtube.com/watch?v=wACD8WEnImo&list=PLT98CRl2KxKHy4A5N70jMRYAROzzC2a6x


#
# It is assumed that you are accessing the Desktop computer in the "DNA Lab" at Aarhus to perform Steps 2 through Step 5.
#


## Step 2 - The basics of Nanopore sequencing and setting up your sequencing run and basecalling.

Tutorial video by Oxford Nanopore Technologies on how to use the MinKNOW software: https://www.youtube.com/watch?v=oCOixfewXo4

Tutorial by me on some more details on how and where to direct file output from the MINKNOW: https://youtu.be/Tr4hCS6DBSQ

If you want to learn more about Nanopore sequencing watch the full Nanopore Masterclass series. Very useful information: https://www.youtube.com/watch?v=MYOSUCgr3Y4&list=PLxpxXZj-IgXwq7AUQ93ek7zcgMjS_E0Rd


## Step 3 - Pulling data from GenomeDK

Video tutorial: https://youtu.be/v9ImfpTUMOE


## Step 4 - Getting you basecalled and aligned data through the DNA Sequencing Pipeline for DNA modifications

Pipeline GitHub with documentation: https://github.com/bernardo-heberle/DCNL_NANOPORE_PIPELINE

Video Tutorial: 



## Description of main output from DNA Sequencing Pipeline for DNA modifications


